src/main/java/io/github/jhipster/online/domain/deserializer/YoRCDeserializer.java

```diff
@@ -106,7 +113,9 @@ public YoRC deserialize(JsonParser jp, DeserializationContext ctxt) throws IOExc
             .hasProtractor(hasProtractor)
             .hasGatling(hasGatling)
             .hasCucumber(hasCucumber)
-            .selectedLanguages(languages);
+            .selectedLanguages(languages)
+            .creationDate(Instant.parse(creationDate));
```

Comment:
Not sure about that. I think it will fail if creationDate is ""
And this change should be done in another PR :)

Reply:
I know but I think was eager to fix this one too. My apologies.

Reply:
can you cherry pick the sonar one and leave this as is? I will use it for a new PR then.

Reply:
> Not sure about that. I think it will fail if creationDate is "" And this change should be done in another PR :)

Resolved. Please check the last commit.